support-frontend/assets/pages/subscriptions-redemption/subscriptionsRedemption.jsx

```diff
@@ -52,7 +52,7 @@ const content = (
     >
       <CheckoutStage
         checkoutForm={<RedemptionForm />}
-        thankYouContentPending={<ThankYouPendingContent {...thankyouProps} />}
+        thankYouContentPending={<ThankYouPendingContent includePaymentCopy={false} {...thankyouProps} />}
         thankYouContent={<ThankYouContent {...thankyouProps} />}
```

Comment:
Do we want to pass this `includePaymentCopy` prop to `< ThankYouContent />` too? If so you could add `includePaymentCopy: false` to the `thankyouProps` object and save having to put an inline prop here.

Reply:
no that one seems to actually be an indirect thankYouContainer.jsx that wraps the actual ThankYouContent and adds a bunch of properties to the map including this one.  The "pending" one seems to directly call the view.  I thought about using the same technique, but it seemed overkill for just a single property, hence why I inlined it.
I'm not sure whether one of them should be renamed to make it clear one is direct and other is a container, or whether I should have made both a container?

Reply:
OK I see what you're saying `<ThankYouContent />` is a wrapper to `thankYouContainer.jsx` and this wrapper already passes the `includePaymentCopy` property so no need to include it here. It seems overkill to replicate it. In which case this is fine as is.